Description: Moves the netty-transport-native-epoll out of the profiles
 since it doesn't work with Gradle (required for the projectreactor package)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/all/pom.xml
+++ b/all/pom.xml
@@ -108,43 +108,6 @@
       </dependencies>
     </profile>
 
-    <!-- The linux profile will only include the native jar for epol to the all jar.
-         If you want to also include the native jar for kqueue use -Puber.
-    -->
-    <profile>
-      <id>linux</id>
-      <activation>
-        <os>
-          <family>linux</family>
-        </os>
-      </activation>
-      <dependencies>
-        <!-- All release modules -->
-        <dependency>
-          <groupId>${project.groupId}</groupId>
-          <artifactId>netty-transport-native-epoll</artifactId>
-          <version>${project.version}</version>
-          <classifier>${jni.classifier}</classifier>
-          <scope>compile</scope>
-          <optional>true</optional>
-        </dependency>
-        <!-- Just include the classes for the other platform so these are at least present in the netty-all artifact -->
-        <dependency>
-          <groupId>${project.groupId}</groupId>
-          <artifactId>netty-transport-native-kqueue</artifactId>
-          <version>${project.version}</version>
-          <scope>compile</scope>
-          <optional>true</optional>
-        </dependency>
-        <dependency>
-          <groupId>${project.groupId}</groupId>
-          <artifactId>netty-resolver-dns-native-macos</artifactId>
-          <version>${project.version}</version>
-          <scope>compile</scope>
-          <optional>true</optional>
-        </dependency>
-      </dependencies>
-    </profile>
     <!-- The mac, openbsd and freebsd  profile will only include the native jar for epol to the all jar.
          If you want to also include the native jar for kqueue use -Puber.
     -->
@@ -361,6 +324,12 @@
     <!-- All release modules -->
     <dependency>
       <groupId>${project.groupId}</groupId>
+      <artifactId>netty-transport-native-epoll</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
       <artifactId>netty-buffer</artifactId>
       <version>${project.version}</version>
       <scope>compile</scope>
